src/shadowbox/server/shared_metrics.spec.ts

```diff
@@ -12,13 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import {DailyFeatureMetricsReport, HourlyConnectionMetricsReport} from 'outline-metrics-server';
```

Comment:
Can't you import from 'outline-metrics-server/model' ?

Reply:
Done.